src/modules/epsr/epsr.h

```diff
@@ -109,7 +109,7 @@ class EPSRModule : public Module
      * Functions
      */
     private:
-    // Target Configuration (determined from target modules)
+    // Target Configuration (determined from target modules) and a function to return this for EPSRManager
```

Comment:
Don't really need this additional comment:
```suggestion
    // Target Configuration (determined from target modules)
```